src/debug_menu.cpp

```diff
@@ -1873,6 +1895,33 @@ void debug()
         case debug_menu_index::TEST_MAP_EXTRA_DISTRIBUTION:
             MapExtras::debug_spawn_test();
             break;
+
+        case debug_menu_index::VEHICLE_BATTERY_CHARGE: {
+
+            optional_vpart_position v_part_pos = here.veh_at( u.pos() );
+            if( !v_part_pos ) {
+                add_msg( m_bad, _( "There's no vehicle there." ) );
+                break;
+            }
+
+            vehicle *const veh = &v_part_pos->vehicle();
```

Comment:
```suggestion
            vehicle &veh = v_part_pos->vehicle();
```
There is no reason to use a pointer here.

Reply:
Fixed.